stable/prometheus-rabbitmq-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 0.1.3
+version: 0.1.4
```

Comment:
I think technically this should `0.2.0`, because you are adding new functionality instead of fixing a bug.

Reply:
Thanks. Sounds reasonable. I'll change it